.gitignore

```diff
@@ -36,3 +36,6 @@ Pipfile
 Pipfile.lock
 composer.lock
 .phpunit.result.cache
+
+pullMagic.sh
```

Comment:
You shouldn't really add your personal files to every bodies .gitignore. You should configure a global .gitignore for just yourself and list these files in it

See:
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c